contracts/ArbitrableDirectory.sol

```diff
@@ -0,0 +1,827 @@
+     */
+    function changeWinnerStakeMultiplier(uint _winnerStakeMultiplier) external onlyGovernor {
+        winnerStakeMultiplier = _winnerStakeMultiplier;
+    }
+
+    /** @dev Change the proportion of arbitration fees that must be paid as fee stake by the party that lost the previous round.
+     *  @param _loserStakeMultiplier Multiplier of arbitration fees that must be paid as fee stake. In basis points.
+     */
+    function changeLoserStakeMultiplier(uint _loserStakeMultiplier) external onlyGovernor {
+        loserStakeMultiplier = _loserStakeMultiplier;
+    }
+
+    /** @dev Change the arbitrator to be used for disputes that may be raised. The arbitrator is trusted to support appeal periods and not reenter.
+     *  @param _arbitrator The new trusted arbitrator to be used in disputes.
+     *  @param _arbitratorExtraData The extra data used by the new arbitrator.
+     */
+    function changeArbitrator(IArbitrator _arbitrator, bytes calldata _arbitratorExtraData) external onlyGovernor {
+        arbitrator = _arbitrator;
+        arbitratorExtraData = _arbitratorExtraData;
+    }
+
+    /** @dev Update the meta evidence used for disputes.
+     *  @param _metaEvidence The meta evidence to be used for future disputes.
+     */
+    function changeMetaEvidence(string calldata _metaEvidence) external onlyGovernor {
+        metaEvidenceUpdates++;
+        emit MetaEvidence(metaEvidenceUpdates, _metaEvidence);
+    }
+
+    // ****************************** //
+    // *   Requests and Challenges  * //
+    // ****************************** //
+
+    /** @dev Make a request to add an organization to the directory. Requires a Lif deposit.
+     *  @param _organization The ID of the organization.
+     */
+    function requestToAdd(bytes32 _organization) external {
+        Organization storage organization = organizationData[_organization];
+        // Get the organization info from the ORG.ID registry.
+        (bool exists,,,,,,, address orgOwner, address director, bool isActive, bool isDirectorshipAccepted) = orgId.getOrganization(_organization);
+
+        require(organization.status == Status.Absent, "Directory: The organization must be either registered or registering.");
+        require(exists, "Directory: Organization not found.");
+        require(orgOwner == msg.sender || (director == msg.sender && isDirectorshipAccepted), "Directory: Only organization owner or director can add the organization.");
+        require(isActive, "Directory: Only enabled organizations can be added.");
+
+        organization.ID = _organization;
+        organization.status = Status.RegistrationRequested;
+        organization.requester = msg.sender;
+        organization.lastStatusChange = now;
+        organization.lifStake = requesterDeposit;
+        require(lif.transferFrom(msg.sender, address(this), requesterDeposit), "Directory: The token transfer must not fail.");
```

Comment:
it is maybe better to use SafeERC20 transfer method

Reply:
It was used in one of the older versions but later we decided that it'd be too excessive and just a simple `require ` for transfer is enough

Reply:
ok